exporter/prometheusremotewriteexporter/README.md

```diff
@@ -66,6 +66,8 @@ The following settings can be optionally configured:
 - `max_batch_size_bytes` (default = `3000000` -> `~2.861 mb`): Maximum size of a batch of
   samples to be sent to the remote write endpoint. If the batch size is larger
   than this value, it will be split into multiple batches.
+- `max_batch_request_parallelism` (default = `5`): Maximum parallelism allowed for a single request bigger than `max_batch_size_bytes`.
+  This configuration is used only when feature gate `exporter.prometheusremotewritexporter.EnableMultipleWorkers` is enabled.
```

Comment:
I think the implementation differs from this documentation.  It seems like this config option will be used if set regardless of the state of the feature gate.

Reply:
@Aneurysm9 is correct.  Please remove this sentence

Reply:
Correct. I didnt review README after all the changes. Please take another look. If it LGTM do let me know to squash and rebase